# HG changeset patch
# User Ben Abbott <bpabbott@mac.com>
# Date 1268439339 18000
#      Fri Mar 12 19:15:39 2010 -0500
# Node ID b3ec24dc305ae97dfcdd6ed765bd6f05555e7dfc
# Parent  a87afd063e7d0401b2eb9f13f808f7ab57b273c1
chol.cc: Increase tolerance for single precision test.

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,12 @@
+2010-03-12  Ben Abbott <bpabbott@mac.com>
+
+	* /DLD-FUNCTIONS/chol.cc: Increase tolerance for single precision test.
+
 2010-03-09  Jaroslav Hajek  <highegg@gmail.com>
 
 	* DLD-FUNCTIONS/lookup.cc (Flookup): Fix deprecated array constructor.
 
 2010-03-09  Jaroslav Hajek  <highegg@gmail.com>
 
 	* ov-intx.h 
 	(OCTAVE_INT_NDARRAY_T::convert_gripe): Remove method.
diff --git a/src/DLD-FUNCTIONS/chol.cc b/src/DLD-FUNCTIONS/chol.cc
--- a/src/DLD-FUNCTIONS/chol.cc
+++ b/src/DLD-FUNCTIONS/chol.cc
@@ -980,17 +980,17 @@ If @var{info} is not present, an error m
 %!                 0.45266  + 0.50020i]);
 %!
 %! R = chol(single(Ac));
 %! 
 %! j = 3; p = [1:j-1, j+1:5];
 %! R1 = cholinsert(R,j,u2); A1 = R1'*R1;
 %! 
 %! assert(norm(triu(R1)-R1,Inf) == 0)
-%! assert(norm(A1(p,p) - single(Ac),Inf) < 1e1*eps('single'))
+%! assert(norm(A1(p,p) - single(Ac),Inf) < 2e1*eps('single'))
 %! 
 */
 
 DEFUN_DLD (choldelete, args, ,
   "-*- texinfo -*-\n\
 @deftypefn {Loadable Function} {@var{R1} =} choldelete (@var{R}, @var{j})\n\
 Given a Cholesky@tie{}factorization of a real symmetric or complex hermitian\n\
 positive definite matrix @w{@var{A} = @var{R}'*@var{R}}, @var{R}@tie{}upper triangular,\n\
